lib/src/style/QlementineStyle.cpp

```diff
@@ -1492,8 +1492,8 @@ void QlementineStyle::drawControl(ControlElement ce, const QStyleOption* opt, QP
           const auto colorize = QlementineStyle::isAutoIconColorEnabled(w);
           const auto& colorizedPixmap = colorize ? getColorizedPixmap(pixmap, fgColor) : pixmap;
           const auto pixmapPixelRatio = colorizedPixmap.devicePixelRatio();
-          const auto pixmapW = pixmapPixelRatio != 0 ? (int) ((qreal) colorizedPixmap.width() / pixmapPixelRatio) : 0;
```

Comment:
Why did you remove the multiplication factor? I guess it was necessary to get the correct size, wasn't it?

Reply:
yes, let me close this PR and create another one